composer.json

```diff
@@ -8,11 +8,17 @@
       "email": "contact@vinaikopp.com",
       "homepage": "http://vinaikopp.com",
       "role": "Developer"
+    },
+    {
+      "name": "Simone Marcato",
+      "email": "info@hevelop.com",
+      "homepage": "https://hevelop.com",
+      "role": "CTO"
     }
   ],
   "require": {
-    "php": "~5.6.0",
-    "magento/module-eav": ">=0.74.0-beta14,<2.0"
+    "php": ">=5.6.0",
```

Comment:
The PHP version requirements should be `"^5.6.0 || ^7.0.0"` so it does not include 7.1 or even newer versions.